build-system/tasks/e2e/describes-e2e.js

```diff
@@ -486,6 +488,12 @@ function describeEnv(factory) {
         }
       });
 
+      before(() => {
+        if (argv.coverage) {
+          istanbulMiddleware = require('istanbul-middleware/lib/core');
```

Comment:
Would we be able to do this require at the top of the file, still guarded by `argv.coverage`? It seems odd to have it down here in a `before`

Reply:
There is precedent for `require` statements inside test code, especially when there is good reason to do it lazily. E.g. Lines 57-58 here.

https://github.com/ampproject/amphtml/blob/703f1139f314330165ac44888f6c54af9d71beae/build-system/tasks/e2e/index.js#L49-L63

In this case, I've moved it to the top inside a conditional, since the location is not critical to the flow of this task.